mantidproject/mantid
# Add Current Save Folder to SANS, change default save option to Memory

**Description of work.**
Added text in the SANS reduction GUI to display the current save location for easier management.
Additionally, changed the default save option from **both** to **memory**.

To keep uniformity across the application, text on the *Sum Runs* tab was changed from "Output Directory" to "Save Directory"

**Report to:** [nobody]

**To test:**

On Interfaces -> SANS -> ISIS SANS v2 experimental, with the runs tab selected text will show "Save Directory: " plus the current path, if selected. Click *Manage Directories* and select a default save directory. This directory change should be reflected in the text.

Additionally, in *Save Options* in the bottom right corner of the GUI, the default radio button should be *Memory*.

Fixes #23957.
Fixes #23968.
---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
